File: Node/ex_node.cpp
#include <ros/ros.h>

int main(int argc, char **argv)

{
	ros::init(argc, argv, "a_node_name"); //#This has to be unique or you can add a parameter ->  ros::init_options::AnonymousName
	ros::NodeHandle nh;
	ROS_INFO("This is a node that called a_node_name");
	ros::Duration(1).sleep();
	ROS_INFO("Exit now");
}

File: Node/ex_node.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
# Author: hbasriaykol

import rospy

if __name__ == '__main__':
    rospy.init_node('a_node_name')
    # This has to be unique or you can add a parameter -> anonymous = True
    rospy.loginfo("This is a node that called a_node_name.")
    rospy.sleep(1)
    rospy.loginfo("Exit now")
